pkg/json.go

```diff
@@ -83,6 +86,32 @@ type JSONScorecardResultV2 struct {
 	Metadata       []string            `json:"metadata"`
 }
 
+// nolint: govet
+type jsonCheckResultV3 struct {
+	Risk     rules.Risk        `json:"risk"`
+	Outcome  finding.Outcome   `json:"outcome"`
```

Comment:
is your intention to simplify checks to pass/fail? Does a check fail when any rule fails?

Reply:
I think that's correct. It's jut a summary of the overall result. Users can parse each rule's result to find out what's passing or not. User can ignore the check's `Outcome` and look at the findings.

Any concern?